tests/components/brother/test_config_flow.py

```diff
@@ -5,30 +5,23 @@
 from brother import SnmpError, UnsupportedModel
 
 from homeassistant import data_entry_flow
-from homeassistant.components.brother import config_flow
 from homeassistant.components.brother.const import DOMAIN
-from homeassistant.const import CONF_HOST, CONF_NAME, CONF_TYPE
+from homeassistant.config_entries import SOURCE_USER, SOURCE_ZEROCONF
+from homeassistant.const import CONF_HOST, CONF_TYPE
 
-from tests.common import load_fixture
+from tests.common import MockConfigEntry, load_fixture
 
-CONFIG = {
-    CONF_HOST: "localhost",
-    CONF_NAME: "Printer",
-    CONF_TYPE: "laser",
-}
+CONFIG = {CONF_HOST: "localhost", CONF_TYPE: "laser"}
 
 
 async def test_show_form(hass):
     """Test that the form is served with no input."""
-    flow = config_flow.BrotherConfigFlow()
-    flow.hass = hass
-
     result = await hass.config_entries.flow.async_init(
-        DOMAIN, context={"source": "user"}
+        DOMAIN, context={"source": SOURCE_USER}
     )
 
     assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
-    assert result["step_id"] == "user"
+    assert result["step_id"] == SOURCE_USER
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4870117)